src/style.css

```diff
@@ -1 +1,71 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  position: relative;
+  margin: 0 10px;
+  display: flex;
+  text-decoration: none;
+  height: 60px;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: rgb(0, 0, 0);
+  cursor: pointer;
+}
+
+.nav__link:hover {
+  color: rgb(0, 172, 220);
+}
+
+.is-active {
+  color: rgb(0, 172, 220);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background: rgb(0, 172, 220);
+  height: 4px;
+}
+
+.nav__item:last-child a {
+  margin-right: 0;
+}
+
+.nav__item:first-child a {
+  margin-left: 0;
+}
```

Comment:
don't stylize tags, use classes instead. And I suggest you use one-direction margins, then you will need to zero only one side margin. 
But we can optimize the code even more by setting margins to only needed elements by using `:not` pseudoclass